tests/playwright/pages/wp-admin-page.ts

```diff
@@ -242,4 +242,11 @@ export default class WpAdminPage extends BasePage {
 		await this.page.goto( '/wp-admin/post-new.php?post_type=page' );
 		await this.closeBlockEditorPopupIfVisible();
 	}
+	async promotionPageScreenshotTest( page: Page, promotionContainer: string, pageUri: string, screenshotName: string ) {
+		await page.goto( `/wp-admin/admin.php?page=${ pageUri }/` );
+		const promoContainer = page.locator( promotionContainer );
+		await promoContainer.waitFor();
+		await expect( promoContainer ).toHaveScreenshot( `${ screenshotName }.png` );
+	}
```

Comment:
1. Add an empty line between the functions
2. use the global page var instead of passing it from the test.
```suggestion
	}

		async promotionPageScreenshotTest( promotionContainer: string, pageUri: string, screenshotName: string ) {
		await this.page.goto( `/wp-admin/admin.php?page=${ pageUri }/` );
		const promoContainer = this.page.locator( promotionContainer );
		await promoContainer.waitFor();
		await expect( promoContainer ).toHaveScreenshot( `${ screenshotName }.png` );
	}
```